src/context/OrderContext.tsx

```diff
@@ -59,9 +47,7 @@ export const LocalOrderProvider: React.FC<Props> = ({ children }) => {
   };
 
   return (
-    <LocalOrderContext.Provider
-      value={{ localOrder, addToLocalOrder, increaseProductQuantity, decreaseProductQuantityOrRemove, removeProduct, setOrderId, removeLocalOrder }}
-    >
+    <LocalOrderContext.Provider value={{ localOrder, addToLocalOrder, removeProduct, setOrderId, removeLocalOrder }}>
```

Comment:
one of the code smells wants this to be wrapped in a memo, which makes sense - function Component() {
  const foo = useMemo(() => ({foo: 'bar'}), []);
  return (
    <SomeContext.Provider value={foo}>
      <SomeComponent />
    </SomeContext.Provider>
  );
}